extra/Android.mk

```diff
@@ -1,12 +1,33 @@
 LOCAL_PATH := $(call my-dir)
 
+# Microg F-Droid repos
+include $(CLEAR_VARS)
+LOCAL_MODULE := microg_repos.xml
+LOCAL_MODULE_TAGS := optional
+LOCAL_MODULE_CLASS := ETC
+LOCAL_MODULE_PATH := $(TARGET_OUT_VENDOR_ETC)/org.fdroid.fdroid
+LOCAL_OVERRIDES_PACKAGES := additional_repos.xml
+LOCAL_SRC_FILES := microg_repos.xml
+include $(BUILD_PREBUILT)
+
+# Bromite F-Droid repos
+include $(CLEAR_VARS)
+LOCAL_MODULE := bromite_repos.xml
+LOCAL_MODULE_TAGS := optional
+LOCAL_MODULE_CLASS := ETC
+LOCAL_MODULE_PATH := $(TARGET_OUT_VENDOR_ETC)/org.fdroid.fdroid
+LOCAL_OVERRIDES_PACKAGES := additional_repos.xml
+LOCAL_SRC_FILES := bromite_repos.xml
+include $(BUILD_PREBUILT)
+
 # additional F-Droid repos
 include $(CLEAR_VARS)
-LOCAL_MODULE := additional_repos.xml
+LOCAL_MODULE := extendrom_additional_repos.xml
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE_CLASS := ETC
 LOCAL_MODULE_PATH := $(TARGET_OUT_VENDOR_ETC)/org.fdroid.fdroid
-LOCAL_SRC_FILES := $(LOCAL_MODULE)
+LOCAL_OVERRIDES_PACKAGES := additional_repos.xml
+LOCAL_SRC_FILES := additional_repos.xml
```

Comment:
why is that needed? can't we just keep `additional_repos.xml`? I also wondering about the LOCAL_OVERRIDES_PACKAGES in that context.. does that mean you want to keep it "backwards"  compatible for those having `additional_repos.xml` specified in their config? If the above works like that it would be ok I just want to understand how that works or what the idea is behind

Reply:
Yes. I wanted it to work for people who want both the MicroG repos **and** the Bromite repos. 

I also think it needed renaming because my builds are picking up MicroG (and F-Droid etc.) from the [`lineageos4microg/android_vendor_partner_gms` repo](https://github.com/lineageos4microg/android_vendor_partner_gms), which includes a `additional_repos.xml` module, and the build complained about the duplication